ring/src/piop/params.rs

```diff
@@ -32,7 +32,13 @@ pub struct PiopParams<F: PrimeField, Curve: SWCurveConfig<BaseField=F>> {
 
 impl<F: PrimeField, Curve: SWCurveConfig<BaseField=F>> PiopParams<F, Curve> {
     pub fn setup(domain: Domain<F>, h: Affine<Curve>, seed: Affine<Curve>) -> Self {
-        let padding_point = crate::hash_to_curve::<Affine<Curve>>(b"w3f/ring-proof/common/padding");
+        let padding_point = {
+            use ark_std::{rand::SeedableRng, UniformRand};
+            use blake2::Digest;
+            let seed = blake2::Blake2b::digest(b"w3f/ring-proof/common/padding");
```

Comment:
Do we want to use a more "neutral" seed here?

Reply:
What do you mean?  An empty string or something?  Nah it's nice the string says something.

Reply:
No I mean something not associated to any organization. Like "ring-proof-padding"